bindata/network/ovn-kubernetes/ovnkube-node.yaml

```
@@ -274,6 +299,9 @@ spec:
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             --inactivity-probe="${OVN_CONTROLLER_INACTIVITY_PROBE}" \
             ${gateway_mode_flags} \
+            {{ if eq .OVN_DPU_MODE "dpu-host" }}
+            --ovnkube-node-mode dpu-host \
```

```suggestion
            --ovnkube-node-mode smart-nic-host \
```
Needed until https://github.com/ovn-org/ovn-kubernetes/pull/2409/files merges

---

Need this before `exec /usr/bin/ovnkube ... ` is called:
```
          node_mgmt_port_netdev_flags=
          if [[ -n "${OVNKUBE_NODE_MGMT_PORT_NETDEV}" ]] ; then
            node_mgmt_port_netdev_flags="--ovnkube-node-mgmt-port-netdev ${OVNKUBE_NODE_MGMT_PORT_NETDEV}"
          fi
```
And then this here:
```suggestion
            --ovnkube-node-mode dpu-host \
            ${node_mgmt_port_netdev_flags} \
```